PINCache/PINDiskCache.m

```diff
@@ -744,9 +742,18 @@ - (id)objectForKeyedSubscript:(NSString *)key
                 PINDiskCacheError(error);
             }
             [self lock];
-          if (!self->_ttlCache) {
-            [self _locked_setFileModificationDate:now forURL:fileURL];
```

Comment:
I thought I had a good reason, but thinking through it more I don't. I'll pull it out into another method and call it from every place, good feedback.